tests/unit/oidc/test_views.py

```diff
@@ -419,13 +419,13 @@ def test_mint_token_from_pending_trusted_publisher_invalidates_others(
 
     user = UserFactory.create()
     pending_publisher = PendingGitHubPublisherFactory.create(
-        project_name="does-not-exist",
+        project_name="DOES-NOT-EXIST",
```

Comment:
Not necessary.  I changed it because it was identical to one of the invalidated projects in the code below this.  It seemed like the code was partially testing project name normalization so made the project name yet another variation that normalization would cause to match.  Will revert though to reduce noise.